fsw/src/to_lab_version.h

```diff
@@ -35,14 +37,19 @@
 #define TO_LAB_MINOR_VERSION 3  /*!< @brief Minor version number */
 #define TO_LAB_REVISION      99 /*!< @brief Revision version number. Value of 99 indicates a development version.*/
```

Comment:
REVISION should be 0 instead of MISSION_REV being 00. Can use to set version string to development vs release.